src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,26 @@
 import React from 'react';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
 
-export const TodoModal: React.FC = () => {
+interface Props {
+  selectedTodo: Todo | null;
```

Comment:
```suggestion
  selectedTodo: Todo;
```
you render this component only when selectedTodo exists